src/dataServices/dynamoDbBundleService.test.ts

```diff
@@ -320,9 +326,11 @@ describe('atomicallyReadWriteResources', () => {
                 startTime: new Date(),
             });
 
+            newResource.ttlInSeconds = Math.floor(Date.now() / 1000 + 60);
+
             // CHECK
-            // transactWriteItem requests is called thrice
-            expect(transactWriteItemSpy.calledThrice).toBeTruthy();
+            // transactWriteItem requests is called 4x
+            expect(transactWriteItemSpy.callCount).toEqual(4);

```

Comment:
We don't, mainly because the test setup code was pretty rough. I'll update the `runUpdateTest` method to take in another bool to be able to toggle whether to include ttlInSeconds.

Long term, I was going to open another issue and do a PR to get a builder pattern going for the test cases. This would allow us to build permutations of test setup a lot easier and with a fluid api. http://natpryce.com/articles/000714.html

Reply:
pushed 2 new test cases for bundle update to test when there is not a ttlInSeconds set

Reply:
Thank you!